web/client-api/src/main/java/io/deephaven/web/client/api/TableTicket.java

```diff
@@ -40,8 +40,9 @@
     private boolean isConnected = true;
 
     public TableTicket(final Uint8Array ticket) {
+        assert ticket.length == 8;
```

Comment:
just fyi; you can drop this file change for cleaner rebase; current head is:
```
    public TableTicket(final Uint8Array ticket) {
        this.ticket = ticket;

        int id = 0;
        for (int ii = 4; ii >= 1; --ii) {
            id = (id << 8) | ticket.getAt(ii).intValue();
        }
        this.exportId = id;
    }
 ```

Reply:
Dropped as part of the rebase.